GGXrdReversalTool.Library/Scenarios/Action/Implementations/PlayReversalAction.cs

```diff
@@ -11,28 +11,28 @@ namespace GGXrdReversalTool.Library.Scenarios.Action.Implementations;
 public class PlayReversalAction : IScenarioAction
 {
     public IMemoryReader? MemoryReader { get; set; }
-    public SlotInput Input { get; set; } = null!;
+    public SlotInput[] Inputs { get; set; } = null!;
```

Comment:
I get red squigglies saying
> CS8936: Feature 'collection expressions' is not available in C# 10.0. Please use language version 12.0 or greater.

I will replace this with
```suggestion
    public SlotInput[] Inputs { get; set; } = {};
```
because this works.